src/frequenz/sdk/actor/_actor.py

```diff
@@ -0,0 +1,212 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Actor model implementation."""
+
+import abc
+import asyncio
+import logging
+
+from ._background_service import BackgroundService
+
+_logger = logging.getLogger(__name__)
+
+
+class Actor(BackgroundService, abc.ABC):
+    """A primitive unit of computation that runs autonomously.
+
+    From [Wikipedia](https://en.wikipedia.org/wiki/Actor_model):
+
+    > An actor as the basic building block of concurrent computation. In response to
```

Comment:
`as` -> `is`? You quoted it a bit in the middle, here `as` no longer makes sense ;)

Reply:
Yeah, good catch. To keep the quote as a quote I changed it to `From Wikipedia an actor is: > [...] the basic building block....`.